Source/EventHorizon/Producer/EventHorizon.cs

```diff
@@ -149,9 +149,14 @@ async Task ProcessEventsThroughEventHorizon()
                         _cancellationTokenSource.Token).ConfigureAwait(false);
                     if (!tryGetStreamEvent.Success)
                     {
+                        var nextPosition = await publicEvents.GetNextStreamPosition(_cancellationTokenSource.Token).ConfigureAwait(false);
```

Comment:
This does what it should - if anything maybe the name of the variable should reflect that it is for all partitions. Since everything else in this file is scoped to a specific partition?

Reply:
Do you have any name suggestions?

Reply:
Hmm, naming is hard. How about something on the lines of `nextEventPositionInAnyPartition`

Reply:
Explicit and mouthy, let's go for that :D